bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -37,48 +24,48 @@ The variables you will need to get to setup the binding are:
 * `<PASSWORD>` The password attached to the above username.
 
 
-### 2. Bridge and Things Configuration
+The binding has the following configuration options:
```

Comment:
I don‘t think this is a good change. What if I want to get data from two different accounts? This was easy when the account configuration was on bridge-level.

Reply:
Hello @J-N-K , yes I thought of it twice before introducing this change. I did it because : 
1°/ It simplified a lot dependencies between objects in the code, and so the overall code architecture
2°/ It avoided to have 4 level of bridges nested on thing side, that I was not very comfortable with (by the way, I do not like a lot all these bridge things that only takes care of being an API intermediate).

I considered it could be an accepted move because : 
1°/ On Netatmo API, one can manage multiple houses with the same account
2°/ I thought that not so much users would be in a situation where they handle multiple Netatmo API accounts
2.1°/ Even if this case appends, it could be managed using a remote openhab instance.

I hope this clarifies my design choice.

Reply:
Without a complete review I was not able to figure out which levels of bridges you need. The documentation seems not to be up to date there. Could you describe for a simple thing (like a weatherstation - former thing would be `netatmo:NAModule1:3e126fb2e1:020000xxxxxx`) what needs to be configured and why you need four levels, where we had two levels before?

Reply:
I do not need 4 levels everywhere, the main move has been that all things are potential bridges (beside the fact some of them will have no childs). This avoided duplicated code between BridgeHandlers and ThingHandlers that existed in the previous version, while most of the modules have a parent module, wether real, wether virtual. With this setup, configuration is simplified as we do not have to store configuration of the parent, being its bridge.

Netatmo API introduced the concept of Home, that can be security related or energy related. Home hosts rooms, or NAPlug, or NAMain. Rooms hosts modules that can be NRV, NATherm1, NAWelcome.